x-pack/plugin/gpu/src/internalClusterTest/java/org/elasticsearch/xpack/gpu/GPUIndexMultiNodeIT.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.gpu;
+
+import org.apache.lucene.tests.util.LuceneTestCase;
+import org.elasticsearch.test.ESIntegTestCase;
+
+@LuceneTestCase.SuppressCodecs("*") // use our custom codec
+@ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.TEST, numDataNodes = 3)
+public class GPUIndexMultiNodeIT extends BaseGPUIndexTestCase {
```

Comment:
What is the goal of this test? How is it different from GPUIndexMixedNodeIT?

Please add comment what we are testing here. This looks to me more like GPUIndexMixedNodeIT